script.js

```diff
@@ -1,4 +1,68 @@
 var main = function (input) {
   var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var words = computerGenResult();
+  if(input == 'scissors' && words == 2){
+    console.log('scissors vs paper')
+    console.log('scissors win')
+    return 'you won!!';
+  }
+  if(input == 'scissors' && words == 3){
+    console.log('scissors v stone')
+    console.log('stone wins')
+    return 'you lost';
+  }
+  if(input == 'scissors' && words == 1){
+    console.log('both words are the same, its a draw')
+    return 'It is a draw!!'
+  }  
+  if(input == 'paper' && words == 1){
+    console.log('paper vs scissors')
+    console.log('paper loses')
+    return 'you lost';
+  }
+  if(input == 'paper' && words == 3){
+    console.log('paper v stone')
+    console.log('paper wins')
+    return 'you win!!'
+  }
+  if(input == 'paper' && words == 2){
```

Comment:
For the draw case, you can just check that `input` is equal to the randomly generated item. No need 3 separate cases for scissors, paper, or stone.